src/main/java/com/ontrack/dto/TaskDTO.java

```diff
@@ -6,12 +6,14 @@ public class TaskDTO {
 	private String name;
 	private String type;
 	private String description;
+	private String taskdate;
 	
-	public TaskDTO(int taskId, String name, String type, String description) {

```

Comment:
I love what you have here in general I feel that you should add a date or starting date to show when the task begin thats why I suggest task date

Reply:
That's a great suggestion! I think a date could be helpful here as well, maybe a start and end date or even just a due date. We'll discuss it and work on implementing it. Thanks!